Tanvir04Khan/01-max-udemy/01-expense-tracker/src/components/Expenses.js

```diff
@@ -0,0 +1,32 @@
+import ExpenseItem from "./ExpenseItem";
+
+import "./Expenses.css";
+
+const Expenses = (props) => {
+  return (
+    <div className="main-content">
+      <ExpenseItem
+        title={props.data[0].title}
+        amount={props.data[0].amount}
+        date={props.data[0].date}
+      />
+      <ExpenseItem
+        title={props.data[1].title}
+        amount={props.data[1].amount}
+        date={props.data[1].date}
+      />
+      <ExpenseItem
+        title={props.data[2].title}
+        amount={props.data[2].amount}
+        date={props.data[2].date}
+      />
+      <ExpenseItem
+        title={props.data[3].title}
+        amount={props.data[3].amount}
+        date={props.data[3].date}
+      />
+    </div>
```

Comment:
can you use `map` here ? if you dont know about that.. still you can try if you know how to render data dynamically in jsx